platform/avr-raven/apps/raven-lcd-interface/raven-lcd.c

```diff
@@ -146,6 +146,9 @@ raven_ping6(void)
     UIP_ICMP_BUF->icmpchksum = 0;
     UIP_ICMP_BUF->icmpchksum = ~uip_icmp6chksum();
 
+#if UIP_CONF_IPV6_RPL
+    rpl_insert_header();
+#endif /* UIP_CONF_IPV6_RPL */
 
     tcpip_ipv6_output();
```

Comment:
yes I'll give this a try. From browing tcpip.c it seems we'll be missing some ext headers until we do as you suggest. For instance TCP traffic etc.